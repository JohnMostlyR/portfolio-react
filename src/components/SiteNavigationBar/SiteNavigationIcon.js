import styled from 'styled-components';
import FontAwesome from 'react-fontawesome';

import mq from '../../styles/templates/mediaQueries';

const SiteNavigationIcon = styled(FontAwesome)`
  display: none !important;

  ${mq.m`
    display: inherit !important;
    transform: rotate(-90deg);
    margin: 0;
  `}
`;

export default SiteNavigationIcon;
