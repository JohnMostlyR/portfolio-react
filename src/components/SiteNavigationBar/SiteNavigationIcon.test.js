import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import SiteNavigationIcon from './SiteNavigationIcon';

describe('SiteNavigationIcon', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<SiteNavigationIcon name={'home'} fixedWidth/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});