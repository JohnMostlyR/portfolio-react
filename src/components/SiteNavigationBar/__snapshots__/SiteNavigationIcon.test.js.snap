// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteNavigationIcon Should render and match the snapshot 1`] = `
.c0 {
  display: none !important;
}

@media (min-width:37.5em) {
  .c0 {
    display: inherit !important;
    -webkit-transform: rotate(-90deg);
    -ms-transform: rotate(-90deg);
    transform: rotate(-90deg);
    margin: 0;
  }
}

<span
  aria-hidden={true}
  className="fa fa-home fa-fw c0"
/>
`;
